cmd/protoc-gen-go-grpc/main.go

```diff
@@ -55,6 +56,7 @@ func main() {
 
 	var flags flag.FlagSet
 	requireUnimplemented = flags.Bool("require_unimplemented_servers", true, "set to false to match legacy behavior")
+	useGenericStreams = flags.Bool("use_generic_streams", false, "set to true to use generic types for streaming client and server objects")
```

Comment:
Should the new flag be marked experimental in the next release of protoc-gen-go-grpc? By experimental, I mean flag string be `use_generic_streams_experimental` and the usage says something like `This flag is EXPERIMENTAL and may be changed or removed in a later release.`

Later when we stabilize the API we can switch the flag's default to true and remove the experimental tags. 

@dfawley, wdyt?

Reply:
That makes sense - the idea would be to delete the flag entirely one release after it is on by default.  So:

1. Add flag, disabled, release.
2. Switch default of flag, release.
3. Remove flag, release.

(All separated by some reasonable amount of time to find bugs/etc.)

Reply:
Makes sense to me! I've updated the name and description of the flag, updated the two places it is referenced in scripts in this repo, and updated the release notes in the PR description.